mpitests/test_native_mcm.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for default qubit preprocessing."""
+import numpy as np
+import pennylane as qml
+import pytest
+from conftest import LightningDevice, device_name
+from mpi4py import MPI
+
+if device_name not in ("lightning.qubit", "lightning.kokkos", "lightning.gpu"):
+    pytest.skip("Native MCM not supported. Skipping.", allow_module_level=True)
```

Comment:
I think we do not need these lines. This test is only for `lightning.gpu` :thinking: